ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml

```diff
@@ -258,41 +258,12 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: rosa-aws-sts-shared-vpc
-- as: aws-rosa-sts-hcp-full-f14
-  cron: 23 15 2,17 * *
+- as: aws-rosa-sts-hcp-sector-prod-critical-f7
+  cron: 29 10 5,14,21,28 * *
   steps:
     cluster_profile: aws-sd-qe
     env:
       CHANNEL_GROUP: candidate
-      OPENSHIFT_VERSION: "4.14"
-    test:
-    - chain: cucushift-installer-check-cluster-health
-    workflow: rosa-aws-sts-hypershift
-- as: aws-rosa-sts-hcp-advanced-full-f14
```

Comment:
Let us keep the same job list as 4.15. In the stable channel, currently, we need to run all of the workflows on the staging OCM env to do the cluster health checking with the 4.14.z openshift version.